lesson 10/styles/style.css

```diff
@@ -0,0 +1,1884 @@
+    background-image: url('../images/fon-triphouse.jpg');
+    background-repeat: no-repeat;
+    background-position: top center;
+    padding-top: 51px;
+}
+
+.triphouse__conteiner {
+    display: grid;
+    width: 100%;
+    margin: 0 auto;
+    padding: 0 104px;
+}
+
+.triphouse__head {
+    width: 100%;
+    height: 46px;
+    grid-template-columns: 1fr 2fr 94px;
+    margin-bottom: 196px;
+}
+
+.triphouse__logo_vector {
+    width: 205px;
+    height: 40px;
+    fill: #3077C6;
+}
+
+.triphouse__text-head {
+    position: relative;
+    vertical-align: top;
+    text-align: end;
+    margin: 8px 0 0;
+    padding-right: 32px;
+    font-family: 'Roboto', sans-serif;
+}
+
+.triphouse__steys:after {
+    display: inline-block;
+    position: absolute;
+    left: 514px;
+    bottom: 0;
+    width: 0;
+    height: 4px;
+    background-color: #F5BD41;
+    content: "";
+    transition: width 0.3s ease-out;
+}
+
+.triphouse__steys:hover:after, .triphouse__steys:focus:after {
+    width: 50px;
+}
+
+.triphouse__attractions:after {
+    display: inline-block;
+    position: absolute;
+    right: 85px;
+    bottom: 0;
+    width: 0;
+    height: 4px;
+    background-color: #F5BD41;
+    content: "";
+    transition: width 0.3s ease-out;
+}
+
+.triphouse__attractions:hover:after, .triphouse__attractions:focus:after {
+    width: 50px;
+}
+
+
+.triphouse__night {
+    width: 30px;
+    height: 30px;
+    margin: 8px 20px 10px 0;
+}
+
+.triphouse__account-circle {
+    width: 40px;
+    height: 40px;
+    background-size: cover;
+    margin: 0 0 6px;
+    fill: #FFFFFF;
+}
+
+.triphouse__menu {
+    display: none;
+    width: 24px;
+    height: 16px;
+    fill: #FFFFFF;
+    margin: 12px 14px 18px;
+}
+
+
+.triphouse__text {
+    font-size: 24px;
+    color: #FFFFFF;
+    text-decoration: none;
+    margin-right: 20px
+}
+
+.triphouse__inscription {
+    font-family: 'Roboto', sans-serif;
+    font-weight: 500;
+    font-size: 50px;
+    color: #FFFFFF;
+    margin-bottom: 100px;
+}
+
+.triphouse__navi {
+    grid-template-columns: 40% 1fr 15px 1fr 2fr;
+    font-family: 'Roboto', sans-serif;
+    font-size: 18px;
+    color: #FFFFFF;
+    margin: 0 -80px;
+}
+
+._t-name {
+    margin-bottom: 24px;
+}
+
+._t-check-in {
+    justify-self: end;
+    margin-right: 5px;
+}
+
+._t-check-out {
+    max-width: max-content;
+    justify-self: start;
+    margin-left: 5px;
+}
+
+.triphouse__form {
+    width: 100%;
+    padding: 0 104px;
+
+}
+
+.triphouse__form-line {
+    grid-template-columns: minmax(10%, 38%) minmax(12%, 0.8fr) 1fr minmax(5%, 16%);
+    height: 65px;
+    padding: 0;
+    margin: 22px 0 118px;
+    border-radius: 8px;
+    background-color: #FFFFFF;
+}
+
+.triphouse__form-city {
+    vertical-align: top;
+    font-family: 'Roboto', sans-serif;
+    text-decoration-color: #383838;
+    font-size: 18px;
+    border: 0;
+    padding-left: 24px;
+    border-radius: 8px;
+    margin: 2px 3px 2px 0;
+    width: auto;
+}
+
+.triphouse__form-city2 {
```

Comment:
bad name. You should give names for your classe like any person  which will be read it could understand for what it.
So form-city2 rename to form--search-field__mob